calculator/parser.py

```diff
@@ -0,0 +1,132 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(num_1, num_2, op):
+    """
+    Evaluates and acts on different binary operators
+
+    Parameters
+    ----------
+    num_1 : integer
+        The first number to be operated on
+    
+    num_2 : integer
+        The second number to be operated on
+    
+    op : char
+        The operator that is going to be applied to the numbers
+
+    Returns
+    -------
+    integer : The final answer after after the nums are operated on
+    """
+
+    if op == "+":
+        return num_1 + num_2
+
+    elif op == "-":
+        return num_2 - num_1
+
+    elif op == "*":
+        return num_1 * num_2
+
+    elif op == "/":
+        return num_2 / num_1
+
+
+def push(e, stack):
+    """
+    Pushes the element (e) to the given stack
+
+    Parameters
+    ----------
+    e : element
+        The element to be pushed on top of the stack
+
+    stack : list
+        The stack that e is going to be pushed onto
+
+    Returns
+    -------
+    list : The stack that is now updated to have e pushed ontop of it
+    """
+    return stack.append(e)
```

Comment:
This does not return the list because `stack.append(e)` does not return the list, it mutates the list that you passed it.

If you want to return a list, you should explicitly `return stack` after the append call.